src/tools/perf/perftest_params.c

```diff
@@ -87,8 +87,10 @@ static void usage(const struct perftest_context *ctx, const char *program)
     printf("     -o             do not progress the responder in one-sided tests\n");
     printf("     -B             register memory with NONBLOCK flag\n");
     printf("     -b <file>      read and execute tests from a batch file: every line in the\n");
-    printf("                    file is a test to run, first word is test name, the rest of\n");
-    printf("                    the line is command-line arguments for the test.\n");
+    printf("                    file is a test to run, the first word is a user-defined \n");
+    printf("                    test name, followed by command-line arguments. For example:\n");
+    printf("                    \n");
```

Comment:
not sure we need this empty line. and if we add it then IMO better also add an empty line after

Reply:
removed extra line